docs/projects/pulsar_lvds/index.rst

```diff
@@ -0,0 +1,346 @@
+.. _pulsar_lvds:
+
+PULSAR_LVDS HDL project
+==================================================================================
+
+Overview
+----------------------------------------------------------------------------------
+
+The :adi:`AD7625`, :adi:`AD7626`, :adi:`AD7960`, :adi:`AD7961` devices are parts
+from ADC LVDS PulSAR family.
+The :adi:`AD7625` / :adi:`AD7626` is a 16-bit, 6 MSPS / 10 MSPS, charge 
+redistribution successive approximation register (SAR) architecture, 
+analog to-digital converter (ADC). SAR architecture allows unmatched performance
+both in noise (92dB SNR) and in linearity (±1 LSB INL / ±0.45 LSB INL). The 
+AD7626 contains a high speed 16-bit sampling ADC, an internal conversion clock,
+and an internal buffered reference. On the CNV edge, it samples the voltage
+difference between IN+ and IN- pins. The voltages on these pins swing in opposite
+phase between 0 V and REF.
+
+The 4.096V reference voltage, REF, can be generated internally or applied
+externally. All converted results are available on a single LVDS self-clocked or
+echoed-clock serial interface reducing external hardware connections. The 
+:adi:`AD7625` / :adi:`AD7626` is available in a 32-lead LFCSP (5mm by 5mm) with
+operation specified from -40°C to +85°C.
+
+The :adi:`AD7960` / :adi:`AD7961` is an 18-bit/16-bit, 5 MSPS, charge
+redistribution successive approximation (SAR), analog-to-digital converter (ADC).
+The SAR architecture allows unmatched performance both in noise and in linearity.
+The :adi:`AD7960` / :adi:`AD7961` contains a low power, high speed, 18-bit/16-bit
+sampling ADC, an internal conversion clock, and an internal reference buffer.
+
+On the CNV± edge, the AD7960 samples the voltage difference between the IN+ and
```

Comment:
This paragraph is also valid for the AD7961

Reply:
Done in V2.